qcodes/dataset/data_set.py

```diff
@@ -1292,7 +1572,7 @@ def _finalize_res_dict_numeric_text_or_complex(
 
         return res_list
 
-    @staticmethod
+    @ staticmethod
```

Comment:
These spaces before staticmethod look odd to me

Reply:
bleh I tried to set auto formatting on vscode to remove white spaces and every time I save it puts a space in from of # and @. Thanks for pointing this out, I'll fix and turn off the auto formatter.

Reply:
That seems less than helpful ;)